crates/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -213,19 +223,26 @@ impl<'a> TransferFunctions for IDLeakAnalysis<'a> {
         bytecode: &Bytecode,
         index: CodeOffset,
         last_index: CodeOffset,
-    ) -> Result<(), ExecutionError> {
-        execute_inner(self, state, bytecode, index)?;
+        _meter: &mut impl Meter,
+    ) -> Result<(), PartialVMError> {
+        let result = execute_inner(self, state, bytecode, index);
+        if let Err(err) = result {
+            return Err(
+                PartialVMError::new(StatusCode::VERIFIER_INVARIANT_VIOLATION)
+                    .with_message(err.to_string()),
+            );
+        };
```

Comment:
Acknowledged this is icky since we can't represent our own SUI `ExecutionError` in `PartialVMError` and resort to string to preserve our `ExecutionError` data. At some point we can work on overcoming that, but will need changes in the Move repo.